storage/rocksdb/rdb_sst_info.cc

```diff
@@ -98,6 +98,8 @@ Rdb_sst_file_ordered::Rdb_sst_file::put(const rocksdb::Slice &key,
   DBUG_ASSERT(m_sst_file_writer != nullptr);
 
   // Add the specified key/value to the sst file writer
+#pragma GCC diagnostic push
```

Comment:
This is a .cc file and thus benign, but I'd prefer to have a matching pop in order for this to survive any future refactoring better